tsbridge/config.go

```diff
@@ -67,9 +67,16 @@ func (c *Config) Metrics() []*Metric {
 	return c.metrics
 }
 
+// ConfigOptions is a set of global options required to initialize configuration.
+type ConfigOptions struct {
+	Filename                    string
+	DatadogMinPointAge          time.Duration
```

Comment:
These two config options have to be passed around quite a lot, and all the plumbing is starting to feel a bit ... unwieldy. I don't think the current approach will scale to many more datadog-specific options. It's not really a comment for this CL, but more a more general "I think you'll need to refactor this soon" feeling. Maybe something like [envconfig](https://github.com/kelseyhightower/envconfig) would help?

Reply:
I've been thinking about this as well. I agree that envconfig, or maybe just moving these into a `global` section of the .yaml config file, would be better.